updatecli/updatecli.d/fleet.yaml

```
@@ -1,4 +1,4 @@
-name: Bump Fleet Helm Charts
+title: Bump Fleet Helm Charts
```

```suggestion
name: Bump Fleet Helm Charts
```

`title` is deprecated in favor of `name`